google-cloud-bigquery/clirr-ignored-differences.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- see http://mojo.codehaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
+
+<differences>
+    <difference>
+        <className>com/google/cloud/bigquery/BigQuery</className>
+        <method>boolean delete(java.lang.String, java.lang.String)</method>
+        <differenceType>7002</differenceType>
+    </difference>
+</differences>
```

Comment:
What this differences represent?

Reply:
because of we have remove method from code
ref : https://www.mojohaus.org/clirr-maven-plugin/